vagrant/base.sh

```diff
@@ -20,38 +20,41 @@ set -ex
 # If you update this, also update tests/docker/Dockerfile
 export KIBOSH_VERSION=d85ac3ec44be0700efe605c16289fd901cfdaa13
 
-if [ -z `which javac` ]; then
-    apt-get -y update
-    apt-get install -y software-properties-common python-software-properties
-    add-apt-repository -y ppa:webupd8team/java
-    apt-get -y update
+path_to_jdk_cache() {
+  jdk_version=$1
+  echo "/var/cache/oracle-jdk8-installer/jdk-${jdk_version}-linux-64.tar.gz"
```

Comment:
we should probably just remove the whole caching piece.

the caching only makes sense in the context of the webupd8team repository and local use of vagrant for system tests. this patch removes use of the old webupd8team installer (which I think is the right long term move given pains we've seen over time). the cache also mostly only made sense when running vagrant locally against virtualbox, where the cached data could be shared against, e.g., 10 VMs by mounting a local directory into each of the VMs, thus avoiding re-downloading the data. I think people have mostly moved to either using local docker builds/tests *or* nightlies against EC2 that don't benefit from the caching and where we should only be building once for an AMI and then launching that N times.

Reply:
I removed the caching piece